.pre-commit-config.yaml

```diff
@@ -0,0 +1,50 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.3.0
+    hooks:
+      # Identify invalid files
+      - id: check-ast
+      - id: check-yaml
+      - id: check-json
+      - id: check-toml
+
+      # git checks
+      - id: check-merge-conflict
+      - id: check-added-large-files
+      - id: detect-private-key
+      - id: check-case-conflict
+
+      # Python checks
+      - id: check-docstring-first
+      - id: debug-statements
+      - id: requirements-txt-fixer
+      - id: fix-encoding-pragma
```

Comment:
Is the pragma `-*- coding: utf-8 -*-` really needed in each file? My experience is that all relevant editors do the right think and I would guess that all not relevant editors do not understand it anyways?

Reply:
I feel like this not really needed if you work, as you said, with a relevant editor like VSCode, PyCharm or even Sublime.